### PR TITLE
I got my intern a KBParadise V60 Type R...

### PR DESCRIPTION
**Pull request by xton**
...as a going-away present and wanted him to have an Xton-special layout. I think it turned out rather well.

### CODE REVIEW

File: keyboards/v60_type_r/keymaps/xtonhasvim/config.h [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* Copyright 2017 REPLACE_WITH_YOUR_NAME
```
Comment (mechmerlin) [inline keyboards/v60_type_r/keymaps/xtonhasvim/config.h]: Please put your name here to get some mad street cred!

File: keyboards/v60_type_r/keymaps/xtonhasvim/keymap.c [hunk 2]
```diff
@@ -0,0 +1,196 @@
+/* Copyright 2017 REPLACE_WITH_YOUR_NAME
```
Comment (mechmerlin) [inline keyboards/v60_type_r/keymaps/xtonhasvim/keymap.c]: More street cred opportunities here.

File: keyboards/v60_type_r/keymaps/xtonhasvim/config.h [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/v60_type_r/keymaps/xtonhasvim/config.h]: Could you replace the first 3 lines here (the ifdef to the include) with `#pragman once`, and remove the endif at the end?

File: users/xtonhasvim/xtonhasvim.h [hunk 4]
```diff
@@ -22,10 +22,8 @@
 
 #define X_____X KC_NO
 
-bool process_record_xtonhasvim(uint16_t keycode, keyrecord_t *record);
-
 enum xtonhasvim_keycodes {
-  DUMMY = SAFE_RANGE,
+  DUMMY = SAFE_RANGE + 20, // give keyboard-specific codes some room
```
Comment (drashna) [inline users/xtonhasvim/xtonhasvim.h]: This doesn't need to be a dummy keycode. You can absolutely use it.

**Comment by xton** [comment]
Fixes done! Thanks for the review @drashna, @mechmerlin

**Comment by drashna** [comment]
Thanks!

**Pull request merged**